### PR TITLE
apollo-gateway: split secrets, go through CDN

### PR DESCRIPTION
**Pull request by pcarrier**
Less cross-continental traffic, better performance, everybody is happier 🎉

### CODE REVIEW

**Comment by trevor-scheer** [comment]
Awesome! Looks like some of the mocks need to be updated and this should be all set.

Starting here:
https://github.com/apollographql/apollo-server/blob/5eef2a663c82dbe5f450806a6429791db117a4e5/packages/apollo-gateway/src/__tests__/integration/nockMocks.ts#L46-L50

File: packages/apollo-gateway/src/loadServicesFromStorage.ts [hunk 1]
```diff
@@ -38,12 +38,12 @@ const urlFromEnvOrDefault = (envKey: string, fallback: string) =>
...
 const urlPartialSchemaBase = urlFromEnvOrDefault(
   envOverridePartialSchemaBaseUrl,
-  'https://storage.googleapis.com/engine-partial-schema-prod/',
+  'https://managed-federation.apollographql.com/',
 );
 
 const urlStorageSecretBase: string = urlFromEnvOrDefault(
   envOverrideStorageSecretBaseUrl,
-  'https://storage.googleapis.com/engine-partial-schema-prod/',
+  'https://managed-federation.apollographql.com/',
```
Comment (glasser) [inline packages/apollo-gateway/src/loadServicesFromStorage.ts:46]: Can we use two different URLs here, even if for now they are backed by the same bucket and CDN?

Comment (pcarrier) [reply packages/apollo-gateway/src/loadServicesFromStorage.ts:46]: @glasser yes, pick them please.

**Comment by glasser** [comment]
Rebased, fixed CHANGELOG conflict, linked to the PR. The test file I fixed was networkRequests.test.ts FYI.

**Comment by trevor-scheer** [comment]
Thank you @pcarrier and @glasser! I'll merge and release this on the `next` tag tomorrow for us to try out before we graduate it to `latest`.

**Pull request merged**